index.js

```diff
@@ -48,6 +48,11 @@ Wemo.prototype.load = function(setupUrl, cb) {
           cb.call(self, device);
         }
       }
+    } else {
+      debug('Error occurred connecting to device at %s', location);
+      if (cb) {
+        cb.call(self, null, err);
```

Comment:
I agree. I only chose to put it at the end so that it would be backwards compatible. I think it's much better long term for it to be consistent with the err as the first param